src/plugins/intel_gpu/src/plugin/ops/dft.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <intel_gpu/plugin/program.hpp>
+#include <intel_gpu/plugin/common_utils.hpp>
+
+#include <ngraph/op/dft.hpp>
+#include <ngraph/op/constant.hpp>
+
+#include <intel_gpu/primitives/dft.hpp>
+
+namespace ov {
+namespace runtime {
+namespace intel_gpu {
+
+static void CreateDFTOp(Program &p, const std::shared_ptr<ngraph::op::v7::DFT> &op) {
+    auto &outShape = op->get_output_shape(0);
+    {
+        auto r = outShape.size();
+        if (r != 5) // only tested on 5 and bfzyx is hardcoded below
+            throw std::runtime_error { "dft v7 output rank is " + std::to_string(r) };
+    }
+    auto axes = dynamic_cast<const ngraph::op::Constant&>(*op->get_input_node_shared_ptr(1)).cast_vector<int32_t>();
+    {
+        auto dataRank = op->get_input_shape(0).size();
+        for (auto &axis : axes)
+            if (axis < 0)
+                axis += dataRank - 1;
```

Comment:
I think sth like "normalizeAxes" already presented in common utils. In any case better to move it to a separate method for readability

Reply:
sth like this I mean
https://github.com/openvinotoolkit/openvino/blob/3bd6eb6dcdeeea1b1381354cff3a9c814a1a77ee/src/core/src/validation_util.cpp#L884

Reply:
used different overload